main.py

```diff
@@ -34,21 +39,41 @@ def main():
     bot.run(os.environ["TOKEN"])
 
 
+def set_activity():
+    with open("configs/activity.json", "r") as f:
+        activity = json.load(f)
+
+    status = activity["status"]
+    name = activity["name"]
+
+    match status:
+        case "playing":
+            activity = Game(name=name)
+        case "listening":
+            activity = Activity(type=ActivityType.listening, name=name)
+        case "streaming":
+            activity = Streaming(name=name, url=activity["url"])
+        case "watching":
+            activity = Activity(type=ActivityType.watching, name=name)
+
+    return activity
+
+
 def setup_logging():
     """
     Set up the loggings for the bot
     :return: None
     """
     formatter = ColoredFormatter(
-        '%(asctime)s %(log_color)s [%(levelname)s] %(message)s',

```

Comment:
Can you make these back to ' so the formats keeps the same?

Reply:
Got it. I will change later.